components/avatar/__tests__/index.test.tsx

```diff
@@ -24,9 +24,7 @@ describe('Avatar', () => {
   })
 
   it('stacked should be work', () => {
-    const avatar = shallow(
-      <Avatar src="/images/avatar.png" stacked />,
-    )
+    const avatar = shallow(<Avatar src="/images/avatar.png" stacked />)
```

Comment:
These documents should not have changed.

Reply:
i ran a prettier / lint (from the CLI) before pushing. I'm guessing this is what changed it. should we keep it since it's suppose to be done?

Reply:
If you get the result of running `prettier`, there is no problem.